docs/configure/upgrading.md

```diff
@@ -6,7 +6,7 @@ The frontend ecosystem is a fast-moving place. Regular dependency upgrades are a
 
 ## Upgrade script
 
-The most common upgrade is Storybook itself. [Storybook releases](https://storybook.js.org/releases) follow [Semantic Versioning](https://semver.org/). We publish patch releases with bug fixes continuously, minor versions of Storybook with new features every few months, and major versions of Storybook with breaking changes roughly once per year.
+The most common upgrade is Storybook itself. [Storybook releases](https://storybook.js.org/releases) follow [Semantic Versioning](https://semver.org/). We publish patch releases with bug fixes continuously, minor versions of Storybook with new features every few months and major versions of Storybook with breaking changes roughly once per year.
```

Comment:
Minor: Our [style guide specifies that we use the Oxford comma](https://www.notion.so/chromatic-ui/Storybook-docs-style-guide-a3f6a66a8b5547388905d3601f7e6ac9?pvs=4#eb224c2cc635449f863b77e60a28448a).

```suggestion
The most common upgrade is Storybook itself. [Storybook releases](https://storybook.js.org/releases) follow [Semantic Versioning](https://semver.org/). We publish patch releases with bug fixes continuously, minor versions of Storybook with new features every few months, and major versions of Storybook with breaking changes roughly once per year.
```